things/api.py

```diff
@@ -476,30 +485,45 @@ def today(**kwargs):
     """
     Read Today's tasks into dicts.
 
-    Note: This might not produce desired results if the Things app hasn't
-    been opened yet today and the yellow "OK" button clicked for new tasks.
+    Note: Some tasks might be highlighted in yellow within the Things app.
+    These tasks have either been scheduled to show up in the today view on that day
+    (or earlier) or are repeating tasks that are scheduled for that day (or earlier).
+    These tasks are also returned by calling this method, altough the internal structure
+    might change after the yellow "OK" button has been clicked for new tasks.
     In general, you can assume that whatever state the Things app was in
     when you last opened it, that's the state reflected here by the API.
 
     See `things.api.tasks` for details on the optional parameters.
     """
     database = pop_database(kwargs)
-    if not database.was_modified_today():  # pragma: no cover
-        print(
-            "[NOTE] The results reflect the state of the Things app "
-            "when it was last run. If the results seem out of date, "
-            "then run the Things app and click the yellow 'OK' button "
-            "to update Today's to-dos and projects.",
-            file=sys.stderr,
-        )
-    return tasks(
+
+    unconfirmed_scheduled_tasks = tasks(
+        start_date="past",
+        start="Someday",
+        index="todayIndex",
+        database=database,
+        **kwargs,
+    )
+
+    unconfirmed_overdue_tasks = tasks(
+        start_date=False,
+        deadline="past",
+        start="Anytime",
+        suppressed=False,
+        database=database,
+        **kwargs,
+    )
+
+    regular_today_tasks = tasks(
         start_date=True,
         start="Anytime",
         index="todayIndex",
         database=database,
         **kwargs,
     )
```

Comment:
Rule: Put most important thing first. Makes it easier to read. Put the definition of `regular_today_tasks` before the others.

Reply:
ok, done